cmd/telemetrygen/pkg/metrics/aggregation_temporality.go

```diff
@@ -1,3 +1,6 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
please move this change to a separate PR

Reply:
done